change/@microsoft-teams-js-87d62e41-c534-43da-9afb-5dda9cff9030.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added 3P cloud storage provider API support",
```

Comment:
please append " to files.ts" to the end of your comment

Reply:
Done!